spec/std/time/span_spec.cr

```diff
@@ -207,6 +207,14 @@ describe Time::Span do
     # TODO check overflow
   end
 
+  it "divides by another Time::Span" do
+    ratio = 20.minutes / 15.seconds
+    ratio.should eq(80.0)
```

Comment:
I think this is fine. Dividing a time unit by a time unit gives something that has no unit.